### ISSUE TITLE
Deque Axe Tests failing

### ISSUE THREAD

**Issue by jsjf93** [tawk/tawk-messenger-react#18]
Hey, I've added the React component to a project and it has a couple of accessibility tests failing.

Steps to reproduce:
- Add Tawk React component to a project
- Run axe Devtools extension or run e2e accessibility tests e.g. @axe-core/playwright

Errors:
- "description": "Ensures buttons have discernible text",
   "help": "Buttons must have discernible text",
    "helpUrl": "https://dequeuniversity.com/rules/axe/4.6/button-name?application=playwright",
- "description": "Ensures <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text",
    "help": "<svg> elements with an img role must have an alternative text",
    "helpUrl": "https://dequeuniversity.com/rules/axe/4.6/svg-img-alt?application=playwright",
    
Please get in touch if more detail is required.
